package.json

```diff
@@ -23,10 +23,10 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "assemble": "^0.4.42",
     "bootstrap": "^3.3.5",
     "browserstacktunnel-wrapper": "^1.4.2",
     "d3-bundler": "^0.2.7",
+    "gray-matter": "^2.0.2",
```

Comment:
Do we actually have a dependency on this or is it left over from before `tiny-ssg`?